usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -587,22 +587,52 @@ def subaward_annotations(filters: dict):
             .values("total"),
             output_field=TextField(),
         ),
-        "prime_award_disaster_emergency_fund_codes": _disaster_emergency_fund_codes(def_codes=def_codes),
-        "prime_award_outlayed_amount_from_COVID-19_supplementals": _outlay_amount_agg_subquery(
-            emergency_fund_group_name="covid_19",
-            def_codes=def_codes,
+        "prime_award_disaster_emergency_fund_codes": Case(
+            When(
+                sub_action_date__gte=datetime.date(2020, 4, 1),
```

Comment:
I feel like this one in particular can be removed because it's not specific to Covid or Infrastructure. It's just about DEFCs in general